mesonwrap/tools/reviewtool.py

```diff
@@ -108,6 +108,8 @@ def __init__(self, project, clone_url, branch, source_branch=None):
         self._branch = branch
         self._source_branch = source_branch or branch
         self.options = ReviewerOptions()
+        self.permitted_files = ['upstream.wrap', 'meson.build', 'readme.txt',
```

Comment:
We don't actually need this variable for the entire class, let's move it into the is_permitted_file() method?

Reply:
I moved it out because if it is there, then the array gets rebuilt every time the function is called. The perf difference is probably minimal, but still. However if you prefer, I can move it back.